travvy/src/components/Profile.vue

```diff
@@ -1,42 +1,54 @@
 <template>  
   <div class="Profile">
      <div class="container">
-        <h2>Profile Information</h2>
-    <label>Email: </label>
-    <input type="text" class="info" v-model="email">
-    <br>
-    <br>
-     <label>Gender: </label>
-    <input type="text" select class="info" v-model="gender">
-    <br>
-    <br>
-    <label>Age: </label>
-    <input type="text" class="info" v-model="age" > 
-    <br>
-    <br>
-    <label>Preferred activity: </label>
-    <input type="text" class="info" v-model="preferred_activity_type"> 
-    <br>
-    <br>
-    <label>Instagram handle: </label>
-    <input type="text" class="info" v-model="instagram_username"> 
-    <br>
-    <br>
-    <ul>
-      <li><a href="#">Edit profile</a></li>
-    </ul>
+        <h2> Hello {{  getUser().Name }}{{ this.$store.state.name}},</h2>
+        <h2> Here is your profile information: </h2>
+        <br>
+        <label>Email: </label>
+        <input type="text" class="info" v-model="email" :placeholder="[[ this.$store.state.userEmail ]]">
+        <br>
+        <br>
+        <label>Gender: </label>
+        <input type="text" select class="info" v-model="gender" :placeholder="[[ this.$store.state.gender ]]">
+        <br>
+        <br>
+        <label>Age: </label>
+        <input type="text" class="info" v-model="age" :placeholder="[[ this.$store.state.age ]]"> 
+        <br>
+        <br>
+        <label>Preferred activity: </label>
+        <input type="text" class="info" v-model="preferred_activity_type" :placeholder="[[ this.$store.state.activity ]]"> 
+        <br>
+        <br>
+        <label>Instagram handle: </label>
+        <input type="text" class="info" v-model="instagram_username" :placeholder="[[ this.$store.state.instagram ]]"> 
+        <br>
+        <br>
+        <ul>
+          <li><a href="#">Edit profile</a></li>
+        </ul>
```

Comment:
Thanks so much for your thorough updates, Lucy! So exciting to finally be able to see the user information being populated onto the profile page through referencing the store.js file. Also, great use of the "placeholder" attribute to hint suggested values to users, I learned about this attribute thanks to your code!